### ISSUE TITLE
Locked rotations issue

### ISSUE THREAD

**Issue by mstenta** [farmOS/farm_grazing#26]
I noticed an issue while giving a demo today. In the rotations table, when you mark a rotation log as done, it locks it so that you can't move it in the rotations list anymore. However, I dragged a rotation up to right below one of the locked ones, clicked save, and somehow it ended up ABOVE the locked rotation.  And I wasn't able to drag it back underneath again after that.

Not sure why that happened, but it's definitely a bug. Maybe there's an issue with the row "weight" for locked rows?  Maybe it doesn't have a weight, or the weight doesn't get changed after it's locked, and somehow the "unlucked" rows can end up with a weight lower than them? Just a guess... something to test...

**Comment by mstenta** [comment]
Looking into this...

**Comment by mstenta** [comment]
I can't seem to replicate this - I may have actually just remembered it wrong when I wrote the description above. I may have marked the second log as done, not the first. This is maybe a different (usability) issue that we want to look into (it's possible to mark later logs done), but that's not the same as this...

I'm going to close this, and assume that I just misinterpreted what I saw. I'll add another issue for the usability piece so we can think that through.
